src/libs/CardUtils.ts

```diff
@@ -72,11 +73,11 @@ function getYearFromExpirationDateString(expirationDateString: string) {
  * @param cardList - collection of assigned cards
  * @returns collection of assigned cards grouped by domain
  */
-function getDomainCards(cardList: Record<string, OnyxTypes.Card>) {
+function getDomainCards(cardList: OnyxCollection<OnyxTypes.Card>) {
     // Check for domainName to filter out personal credit cards.
     // eslint-disable-next-line you-dont-need-lodash-underscore/filter
-    const activeCards = lodash.filter(cardList, (card) => !!card.domainName && (CONST.EXPENSIFY_CARD.ACTIVE_STATES as ReadonlyArray<OnyxTypes.Card['state']>).includes(card.state));
-    return lodash.groupBy(activeCards, (card) => card.domainName);
+    const activeCards = lodash.filter(cardList, (card) => !!card?.domainName && (CONST.EXPENSIFY_CARD.ACTIVE_STATES as ReadonlyArray<OnyxTypes.Card['state']>).includes(card.state));
```

Comment:
I think it should work this way, could you check please?
```suggestion
    const activeCards = Object.values(cardList ?? {}).filter((card) => !!card?.domainName && CONST.EXPENSIFY_CARD.ACTIVE_STATES.some(state => state === card.state));

```